### DISCUSSION TITLE
NUXT_VITE_NODE_OPTIONS not set / Missing /manifest ?

### DISCUSSION THREAD

**Discussion by toooni** [nuxt/nuxt#16826] [Questions] [answered]
Hi.
I am currenlty on `3.0.0-rc.6` with a pretty big project. Since `3.0.0-rc.7` I am having issues updating because of some issues. Now I thought I'd try the new `3.0.0-rc.11` and am still facing some problems :(

1. NUXT_VITE_NODE_OPTIONS
> Cannot read properties of undefined (reading 'startsWith') at ./node_modules/.pnpm/@nuxt+vite-builder@3.0.0-rc.11_zkwfn7g5d4psjy7k4eo53b6z2i/node_modules/@nuxt/vite-builder/dist/runtime/vite-node-shared.mjs:8:34

This happens because `NUXT_VITE_NODE_OPTIONS` is not set - but I can't find a related issue here on github. I've added the env var `NUXT_VITE_NODE_OPTIONS={"baseURL": ""}` which seems to work. Maybe somebody knows why this is needed?

2. After this I am getting the same error I've had since trying with `rc.7`
> Invalid URL () at async $fetchRaw2 (./node_modules/.pnpm/ohmyfetch@0.4.19/node_modules/ohmyfetch/dist/shared/ohmyfetch.c2a48baf.mjs:132:20)

When debugging this issue it seems that it wants to fetch `/manifest` (no file extension, just /manifest) which does not exist. Should it?

Does anyone have an idea what I could be doing wrong?

**Comment by toooni** [comment, marked as answer]
I've found my mistake. Not sure if it once was normal, or I made the mistake a long time ago, but to have a separate `vite.config.js` doesn't work anymore. I've moved everything from it to the `vite` parameter from `nuxt.config.js`.

**Reply by qbrao** [reply]
I also encountered the same problem, how did you solve it later?

**Reply by toooni** [reply]
As I mentioned, I've moved everything which was in `vite.config.js` into `nuxt.config.js` into the vite config.
